presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestDetermineJoinDistributionType.java

```diff
@@ -623,6 +623,38 @@ public void testReplicatesWhenNotRestricted()
                         values(ImmutableMap.of("B1", 0))));
     }
 
+    @Test
+    public void testChoosesLeftWhenCriteriaEmpty()
+    {
+        int aRows = 1_000__00;
+        int bRows = 1_0;
+        assertDetermineJoinDistributionType(new CostComparator(75, 10, 15))
```

Comment:
Lets add a test that swaps the sides, and a case when the original order is preserved because of the broadcast limit

Reply:
this test does swap the sides. I'll add a test for keeping the original join order though.

Reply:
updated